lib/house.rb

```diff
@@ -0,0 +1,75 @@
+class House
+  attr_reader :string_price,
```

Comment:
We don't need a method called string_price, so this doesn't need to be an attr_reader.